tox.ini

```
@@ -2,11 +2,31 @@
 envlist= py37
 
 [testenv]
+passenv =
+    USE_UTILS2
```

I would recommend making this more descriptive as "use utils" is somewhat vague.
```suggestion
    USE_TEST_UTILS
```